numba/cpython/setobj.py

```diff
@@ -1097,6 +1109,33 @@ def _copy_payload(self, src_payload):
 
         return builder.load(ok)
 
+def _imp_dtor(context, module, set_type):
+    """Define the dtor for set
+    """
+    llvoidptr = context.get_value_type(types.voidptr)
+    llsize = context.get_value_type(types.uintp)
+    fnty = ir.FunctionType(
+        ir.VoidType(),
+        [llvoidptr, llsize, llvoidptr],
```

Comment:
When we were reviewing this code we found determining the arguments confusing, so it might help to explain what the arguments are when you define the function type.